### YamlMime:UniversalReference
api_name: []
items:
- children:
  - azure.mgmt.batchai.models.batch_ai_error.BatchAIError
  fullName: azure.mgmt.batchai.models.batch_ai_error
  langs:
  - python
  module: azure.mgmt.batchai.models.batch_ai_error
  name: batch_ai_error
  source:
    id: batch_ai_error
    path: azure-mgmt-batchai\azure\mgmt\batchai\models\batch_ai_error.py
    remote:
      branch: master
      path: azure-mgmt-batchai\azure\mgmt\batchai\models\batch_ai_error.py
      repo: https://github.com/Azure/azure-sdk-for-python.git
    startLine: 0
  type: module
  uid: azure.mgmt.batchai.models.batch_ai_error
references:
- fullName: azure.mgmt.batchai.models.batch_ai_error.BatchAIError
  isExternal: false
  name: BatchAIError
  parent: azure.mgmt.batchai.models.batch_ai_error
  uid: azure.mgmt.batchai.models.batch_ai_error.BatchAIError
